cmd/cstorvolumeclaim/controller.go

```diff
@@ -282,14 +281,13 @@ func (c *CVCController) distributePendingCVRs(
 	cvc *apis.CStorVolumeClaim,
 	cv *apis.CStorVolume,
 	service *corev1.Service,
-	class *storagev1.StorageClass,
 ) error {
 
-	desiredReplicaCount, err := c.getPendingCVRCount(cvc, class)
+	desiredReplicaCount, err := c.getPendingCVRCount(cvc)
```

Comment:
is this better to be called as 'pendingReplicaCount'.. same/similar variable name in distributeCVRs

Reply:
done